AugurProject/augur-ui
# Sell Complete Sets Button update

added babel-polyfill to fix ledger issue for now, updated sell complete sets button to disable on transaction prompt and only re-enable on error, removed a console log in performance chart

to test:

make a market with an account and put liquidity orders of equal quantity on either side of book. sign all transactions, switch to another account, go to market, buy up the book. Switch back to original account, hit the sell complete sets button. Notice it greys out and is disabled. Reject signing. Note that the button becomes un-disabled and is clickable, click it again. Sign Transaction. Note that the sell complete sets button stays disabled until the sets are sold or there is an error with the tx.

Closes: https://github.com/AugurProject/augur/issues/604
Closes: https://github.com/AugurProject/augur/issues/507
